modules/slacker-core/src/main/java/com/labs2160/slacker/core/engine/Trigger.java

```diff
@@ -0,0 +1,78 @@
+package com.labs2160.slacker.core.engine;
+
+import com.labs2160.slacker.api.Configurable;
+import com.labs2160.slacker.api.RequestHandler;
+import com.labs2160.slacker.api.SlackerRequest;
+import com.labs2160.slacker.api.SlackerResponse;
+
+import java.util.concurrent.Future;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.Executors;
+import java.util.TimerTask;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Trigger implements Configurable {
+
+    private static final Logger logger = LoggerFactory.getLogger(Trigger.class);
+    private final ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(1);
+
+    private String name;
+    private String interval;
+    private Workflow wf;
+    private String[] requests;
+
+    public Trigger(String name, String interval, Workflow wf, String requests) {
+        this.name = name;
+        this.interval = interval;
+        this.wf = wf;
+        this.requests = requests.split(";");
+    }
+
+    @Override
+    public void setConfiguration(Properties config) {
+        // do nothing
+    }
+
+    public void start(RequestHandler handler) {
```

Comment:
This logic should be handled in the engine (or better, a trigger scheduler delegate that the engine oversees).
This will make the Trigger class a simple value object that holds configuration.
Also, we can switch out scheduler implementations a lot easier.